components/sendgrid/actions/get-contact-lists/get-contact-lists.js

```diff
@@ -0,0 +1,44 @@
+const sendgrid = require("../../sendgrid.app");
+const validate = require("validate.js");
+
+module.exports = {
+  key: "sendgrid-get-contact-lists",
+  name: "Get Contact Lists",
+  description: "Allows you to get details of your contact lists.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    sendgrid,
+    numberOfLists: {
+      type: "integer",
+      label: "Number of Lists",
+      description: "The number of contact lists to return.",
+    },
+  },
+  async run() {
+    const constraints = {
+      numberOfLists: {
+        presence: true,
```

Comment:
We should also verify that this number is positive, right? At least so that the user gets a proper error message if they provide the wrong value.

Reply:
I added a validation method for positive ranges.